test_integration/lib/provisioning.dart

```diff
@@ -94,36 +81,9 @@ Future<Map<String, dynamic>> getTokenRequest() async {
   final r = await const RetryOptions(
     maxAttempts: 5,
     delayFactor: Duration(seconds: 2),
-  ).retry(() => http.get(Uri.parse(authURL)));
+  ).retry(() => http.get(Uri.parse(_authURL)));
   print('tokenRequest from tokenRequest server: ${r.body}');
   return Map.castFrom<dynamic, dynamic, String, dynamic>(
     jsonDecode(r.body) as Map,
   );
 }
-
-Future<Map<String, dynamic>> getTokenDetails(
-  String keyName,
-  String keySecret, [
-  String prefix = '',
-]) async {
-  final stringToBase64 = utf8.fuse(base64);
-  final encoded = stringToBase64.encode('$keyName:$keySecret');
-  final r = await const RetryOptions(
-    maxAttempts: 5,
-    delayFactor: Duration(seconds: 2),
-  ).retry(() => http.post(
-        Uri.parse(tokenDetailsURL(keyName, prefix)),
-        headers: {
-          'Authorization': 'Basic $encoded',
-          'Content-Type': 'application/json',
-        },
-        body: json.encode({
-          'keyName': keyName,
-          'timestamp': DateTime.now().millisecondsSinceEpoch,
-        }),
-      ));
-  print('tokenDetails from server: ${r.body}');
-  return Map.castFrom<dynamic, dynamic, String, dynamic>(
-    jsonDecode(r.body) as Map,
-  );
-}
```

Comment:
Now that this content is moved into `test_integration`, we can remove `ably_flutter_example` from `test_integration/pubspec.yaml`

Reply:
Awesome, thanks for catching that. 92b0809afeebef2a6f0876bc2b1ba06cc918c76b